lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1103,6 +1103,17 @@ sub custom_module {
     $self->account_result_size('custom module ' . $module->test->name, $parser->write_output($self->result_dir));
 }
 
+sub job_module {
```

Comment:
I'd call this `job_module_by_name` use the existing relation to implement it, e.g. `$self->modules->search({name => $name})->first`.

Reply:
Fair point.